go/consensus/tendermint/full/light.go

```diff
@@ -54,9 +50,18 @@ func (t *fullService) GetValidatorSet(ctx context.Context, height int64) (*conse
 		return nil, consensusAPI.ErrVersionNotFound
 	}
 
+	protoVals, err := vals.ToProto()
+	if err != nil {
+		return nil, fmt.Errorf("tendermint: failed to marshal validators: %w", err)
+	}
+	meta, err := protoVals.Marshal()
+	if err != nil {
+		return nil, fmt.Errorf("tendermint: failed to marshal validators: %w", err)
```

Comment:
Maybe distinguish from the previous error case somehow so I don't need to hunt down which line number it was on?